launcher/ui/dialogs/ModDownloadDialog.cpp

```diff
@@ -41,7 +42,7 @@ ModDownloadDialog::ModDownloadDialog(const std::shared_ptr<ModFolderModel> &mods
     auto OkButton = m_buttons->button(QDialogButtonBox::Ok);
     OkButton->setDefault(true);
```

Comment:
The `Ok` button should be disabled if no mod was selected. Right now it just shows the message box telling me it's going to download nothing.